Slim/Control/Queries.pm

```diff
@@ -832,7 +833,10 @@ sub albumsQuery {
 
 			$tags =~ /l/ && $request->addResultLoop($loopname, $chunkCount, 'album', $construct_title->());
 			$tags =~ /y/ && $request->addResultLoopIfValueDefined($loopname, $chunkCount, 'year', $c->{'albums.year'});
-			$tags =~ /j/ && $request->addResultLoopIfValueDefined($loopname, $chunkCount, 'artwork_track_id', $c->{'albums.artwork'}) if ($c->{'albums.artwork'} || '') !~ /^https?:/;;
+			if ($tags =~ /j/) {
+				$c->{'albums.artwork'} = $c->{'tracks.coverid'} if $c->{'tracks.coverid'};
```

Comment:
Please use a helper variable rather than re-assigning a given value. `$c` should not be modified. Even if it worked today, code added later and (correctly) assuming that `$c` was the data from the database would fail.